packages/ui/react-ui-editor/src/components/TextEditor/TextEditor.stories.tsx

```diff
@@ -37,11 +37,11 @@ const hyperLinkTooltip = () =>
     );
   });
 
-const Story = () => {
-  const [item] = useState({ text: new TextObject(text) });
+const Story = ({ automerge }: { automerge?: boolean }) => {
```

Comment:
this is kinda awkward, if there's common stuff I'd factor it out to a separate component

Reply:
I've tried using storybook's controlls for this, but got stuck on Story's vs component's prop types missmatch. Can we do a session together where you show me the best prcatices for using storybooks?